### DISCUSSION TITLE
resolved by kernel PR59

### DISCUSSION THREAD

**Discussion by nilesh-dryad** [jefftenney/LPTIM-Tick-U5#2] [General]
I saw this comment in READ ME. I don't get it.
LED is blinking twice particularly in that case. That means error in default tickless mode. Are this comment and error related?

**Comment by jefftenney** [comment]
Yes they are related.  There are some minor errors in the default tickless mode.  Those errors will cause the LED to blink twice in "Test 3".  Those errors are resolved in kernel PR #59, which has not yet been merged.

**Reply by nilesh-dryad** [reply]
That's good to hear. I will take a look in that branch.

**Reply by jefftenney** [reply]
Note that if you use lptimTick.c, it replaces the default tickless mode and therefore eliminates the minor errors in the default tickless mode.  If you still are interested in the fixes to the default tickless mode, you can follow the pull request [here](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/59).  The FreeRTOS team plans to merge the pull request but it's probably a low priority.

**Reply by nilesh-dryad** [reply]
Thanks for letting me know. I also managed to get default tickless mode on U5 series on my own using lptim but it only works with simple task. When I added lwip with FreeRTOS, timing just went crazy. vTaskDelay of 5 seconds is taking more than 1 minutes and then I came to know about this repo therefore I am going to try this repo with LWIP and I will post my result here if it is successful. But it is also low priority as of now for me.
